spec/controllers/admin/users_controller_spec.rb

```diff
@@ -219,7 +219,7 @@
         it 'can add a user to a group' do
           post(:update, params: { id: user.id, user: { group_ids: [admin_group.id, user_group.id] } })
           user.reload
-          expect(user.groups.pluck(:name)).to eq %w[administrators users]
+          expect(user.groups.pluck(:name).sort).to eq %w[administrators users]
```

Comment:
For some reason, this test started failing with the second new test above (`doesn't reset expire_after when changing HTML title`)